doc/dev/repo-access.rst

```diff
@@ -6,7 +6,7 @@ Special branches
 
 * ``master``: current tip (integration branch)
 * Release branches (for example ``luminous``) corresponding to the releases
-  listed at :doc:`releases <../releases>`
+  listed at :doc:`releases <../releases/index>`
```

Comment:
please don't use relative paths, try to use the `:ref:` directive so that when the path changes (like it is today) there is no need to update the path.

The usage of "ref" would mean that a tag will need to be in place. This is an example of setting a ref (above a title) and then consuming them in text:

    .. _ceph-volume-systemd:
    
    systemd
    =======
    As part of the activation process (either with :ref:`ceph-volume-lvm-activate`
    or :ref:`ceph-volume-simple-activate`), systemd units will get enabled that
    will use the OSD id and uuid as part of their name. These units will be run
    when the system boots, and will proceed to activate their corresponding
    volumes via their sub-command implementation.

Reply:
thanks fixing